packages/editor/src/components/collab-sidebar/style.scss

```diff
@@ -18,13 +18,17 @@
 		position: relative;
 		padding: $grid-unit-20;
 		border-radius: $radius-large;
-		border: 1px solid $gray-300;
+		border: 1.5px solid $gray-300;
```

Comment:
Why add a half pixel here?

Reply:
not strong opinion here, We could change it to 1px or 2px, I updated it to make it same as active/focused state border to prevent layout shift on board active/focus state.